table.csv

```diff
@@ -122,6 +122,7 @@ p384-pub,                       key,            0x1201,         P-384 public Key
 p521-pub,                       key,            0x1202,         P-521 public Key
 ed448-pub,                      key,            0x1203,         Ed448 public Key
 x448-pub,                       key,            0x1204,         X448 public Key
+ed25519-priv,                   key,            0x1300,           Ed25519 private key
```

Comment:
very minor nit - 2 spaces need to be removed from the start of the description to make it line up

Reply:
Oops, sorry.

I force pushed the change to fix that.